README.md

```diff
@@ -1 +1,37 @@
-# code-runner
\ No newline at end of file
+Code Runner: An Environment to run your code
+===================================================
+Code Runner is a web application to run code snippet or code files on a docker environment with code specific configurations.
```

Comment:
We don't run code snippets do we? Code snippets are not actually runnable.

Reply:
Changed